dropshot/src/handler.rs

```diff
@@ -1022,6 +1026,7 @@ impl<T: JsonSchema + Serialize + Send + Sync + 'static> HttpTypedResponse
 {
     type Body = T;
     const STATUS_CODE: StatusCode = StatusCode::ACCEPTED;
+    const DESCRIPTION: &'static str = "successful operation";
```

Comment:
I think this one typically just means "successfully enqueued operation" or the like.